src/core/lib/transport/call_filters.h

```diff
@@ -783,7 +834,7 @@ struct ChannelDataDestructor {
 // in-flight calls.
 struct StackData {
   // Overall size and alignment of the call data for this stack.
-  size_t call_data_alignment = 0;
+  size_t call_data_alignment = 1;
```

Comment:
0 is an invalid alignment value, whereas 1 indicates any alignment is cool -- this becomes important because we need to divide by alignment occasionally